app/controllers/lab_tests_controller.rb

```diff
@@ -33,18 +35,17 @@ def edit
 
   # POST /lab_tests or /lab_tests.json
   def create
-    @lab_test = current_user.lab_tests.build(lab_test_params)
+    build_lab_test
```

Comment:
Don't call it explicitly, user before_action instead, as it was [there](https://github.com/Code-the-Dream-School/health-keeper/blob/main/app/controllers/lab_tests_controller.rb#L5) before.